README.md

```diff
@@ -32,6 +32,31 @@ There is:
 
 Please note, much of this material remains a work in progress!
 
+## Devcontainer
+
+The repository includes the option to use [Visual Studio Code remote
+containers](https://code.visualstudio.com/docs/remote/containers), the
+development container includes all the necesary dependencies alongside helpful
+VS Code extension and settings. For more information on how to use it, refer to
+[esp-rs-devcontainer](https://github.com/SergioGasquez/esp-rs-devcontainer).
+> If using Podman, follow along the [Podman setup section](https://github.com/SergioGasquez/esp-rs-devcontainer#optional-podman).
+
+Developing in an online dev enviroment is also available with [Gitpod](https://www.gitpod.io/):
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/github.com/SergioGasquez/espressif-trainings/tree/main)
+
+When using an online Gitpod enviroment, flashing and monitoring can be done
+with online tools, for more information have a look at [Flash](https://github.com/SergioGasquez/esp-rs-devcontainer#adafruit-esptool)
+and [Monitor](https://github.com/SergioGasquez/esp-rs-devcontainer#online-serial-monitor)
+sections of [esp-rs-devcontainer](https://github.com/SergioGasquez/esp-rs-devcontainer) repository.
+
+### Wokwi Simulator
+The devcontainer includes the option of simulating the exercises with [Wokwi](https://wokwi.com/).
+In order to build and run a Wokwi simulation, the task `Build current project and run Wokwi simulation` 
+can be used, it must be invoked with the `main.rs` of the project we want to 
+focused in VS Code. The task will generate a binary file for the application and 
+run Wokwi simulation, this task can also be run with `Ctrl-Shift-B` or `Cmd-Shift-B`.
```

Comment:
The alternative is to open the command palette using `Ctrl-Shift-P` to execute the "Tasks: Run Build Task" or via "Terminal -> Run Task" menu.